addons/l10n_latam_check/wizards/l10n_latam_payment_mass_transfer.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, api, fields, _
+from odoo.exceptions import UserError
+
+
+class L10nLatamPaymentMassTransfer(models.TransientModel):
+    _name = 'l10n_latam.payment.mass.transfer'
+    _description = 'Checks Mass Transfers'
+
+    payment_date = fields.Date(
+        string="Payment Date",
+        required=True,
+        default=fields.Date.context_today,
+    )
+    destination_journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        string='Destination Journal',
+        domain="[('type', 'in', ('bank', 'cash')), ('company_id', '=', company_id), ('id', '!=', journal_id)]",
+    )
+    communication = fields.Char(
+        string="Memo",
+    )
+    journal_id = fields.Many2one(
+        'account.journal',
+        compute='_compute_journal_company'
+    )
+    company_id = fields.Many2one(
+        'res.company',
+        compute="_compute_journal_company"
+    )
+    check_ids = fields.Many2many(
+        'account.payment',
+    )
+
+    @api.depends('check_ids')
+    def _compute_journal_company(self):
+        journal = self.check_ids.mapped("l10n_latam_check_current_journal_id")
+        if len(journal) != 1:
+            raise UserError(_("All selected checks must be on the same journal and on hand"))
+        outbound_pay_method_line = journal._get_available_payment_method_lines('outbound').filtered(
+            lambda x: x.code == 'out_third_party_checks')
+        if not outbound_pay_method_line:
+            raise UserError(_("The journal '%s' don't the payment method to transfer checks"))
```

Comment:
Check the sentence

Reply:
Fixed, check now please
@william-andre

Reply:
It's better but it doesn't include the translations